### ISSUE TITLE
A11y_ML .NET Model Builder_ Data_ Screen Reader: Narrator is not reading the name for the combo boxes "Column to predict", "User Column" and "Item Column"

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1119]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev2; #Benchmark; #Visual Studio; #Screen Reader; #Narrator; #WCAG1.3.1;

### Environment Details:
Microsoft Visual Studio 2019 Preview Version 16.8.0 Preview 6.0

### Screen Readers:
Narrator; 

**NVDA Behavior:** NVDA is reading the label properly

### Note: 
Same issue is observed in "Text Classification" and "Value Prediction" scenarios. Narrator is not reading the label for "Column to Predict" combo box

### Repro Steps:

1.Start Screen Reader
2.Open Visual Studio Enterprise 2019 Preview 
3. Create a new .Net Core Console App (C#) 
4. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
5. Tab till "Recommendation" scenario and select the scenario
6. Tab till "Next Step" button and press enter
7. Select the data source type and upload the file
8. Tab till "Column to predict", "User Column" and "Item Column" combo boxes and verify how narrator is reading the combo boxes

### Actual Result:
Narrator is not reading the name for the combo boxes "Column to predict", "User Column" and "Item Column"

### Expected Result:
Narrator should read the name for the combo box

### User Impact:
Screen Reader users will not get to know about the combo box if screen reader doesn't read the name of the combo box

### Attachment for Reference

[1119_ML .NET Model Builder_ Narrator is not reading the label for combo box.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5607172/1119_ML.NET.Model.Builder_.Narrator.is.not.reading.the.label.for.combo.box.pptx)

[1119_ML .NET Model Builder_ NVDA is  reading the label for combo box.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5607230/1119_ML.NET.Model.Builder_.NVDA.is.reading.the.label.for.combo.box.pptx)

**Comment by SaisreeDandamudi** [comment]
Verified the bug in the given build and the issue is fixed. Hence closing the bug
Please find the below screen shots for reference
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.9.0 Preview 4.0 [30917.4.main]
![Screenshot (53)](https://user-images.githubusercontent.com/51435580/108031901-9c333780-7057-11eb-8947-78929c1e5517.png)

**Comment by poojapatil36** [comment]
Github tag: #Closed;
